plugins/postcss-stepped-value-functions/test/basic.expect.css

```diff
@@ -41,11 +41,17 @@
 	right: 8px;
 }
 
+.unit {
+	top: 37.79528px;
+	bottom: 940px;
+	left: 0.035s;
+}
+
 .invalid {
 	font-size: mod(18px, 5rem);
 	width: rem(var(--foo), 5px);
 	padding: mod(Fo0px, 5px);
-	left: mod(1px, 0px);
+	left: calc(NaN * 1px);
```

Comment:
This is a `NaN` value that still preserves the original unit.
It has no use, but it is technically correct.